cogs/moveusers.py

```diff
@@ -0,0 +1,13 @@
+import discord
```

Comment:
Let's take out this commit from this PR. This should go into a new branch for a new feature for moving users since this is not for auto assigning roles. This PR is getting too big otherwise if we're adding in multiple cogs at a time.

Reply:
ok